domino_data/_feature_store/sync.py

```diff
@@ -110,14 +111,23 @@ def update_feature_views(commit_id: str, repo_path: str) -> None:
                 Entity(name=x, join_key=y.name, value_type=str(y.dtype))
                 for x, y in zip(fv.entities, fv.entity_columns)
             ],
-            features=[Feature(name=f.name, dtype=str(f.dtype)) for f in fv.features],
+            features=[
+                Feature(name=f.name, dtype=str(f.dtype), tags=FeatureTags.from_dict(f.tags))
+                for f in fv.features
+            ],
             ttl=UNSET if fv.ttl is None else int(fv.ttl.total_seconds() * 1000),
             tags=FeatureViewRequestTags.from_dict(fv.tags),
+            description=fv.description,
```

Comment:
description isn't required field when making feature view, what if fv doesn't have a description?

Reply:
See my above comments. This field is am empty string if not provided when initializing the feature view.